ui/src/plot.tsx

```diff
@@ -1112,7 +1112,10 @@ export const
           })
           currentChart.current = chart
           chart.data(data)
-          model.plot.marks.forEach(({ interactive = true }, idx) => chart.geometries[idx].customInfo({ interactive }))
+          // filter out annotation marks
+          model.plot.marks.filter(({ x, y }) => x == null && y == null).forEach(({ interactive = true }, idx) =>
+            chart.geometries[idx].customInfo({ interactive })
+          )
```

Comment:
It seems like this is logically correct, but took me a while to wrap my head around whether the indexes are indeed correct. Let's go for something like

```suggestion
          model.plot.marks.forEach(({ x, y, interactive = true }, idx) => {
            if(isGeometry(x,y)) chart.geometries[idx].customInfo({ interactive })
          })
```
With a helper function (with proper types)
```ts
const isGeometry = (x,y) => x == null || y === null
```

And also refactor the existing `makeMarks` function to use the helper one so that the it's clear what's going on.

Reply:
No problem. I totally understand it. My original implementation was almost the same when I've realised the indexes are wrong. So I came up with the filter solution to keep indexes correct.